internal/handlers.go

```diff
@@ -142,6 +142,30 @@ func (l *Logical) handleClear(_ *QueueState, req *Request) {
 	close(req.ReadyCh)
 }
 
+func (l *Logical) handleReload(state *QueueState, req *Request) {
+	_ = req.Request.(*types.ReloadRequest)
```

Comment:
This is an ineffectual assignment, it only verifies the request type is `types.ReloadRequest`.